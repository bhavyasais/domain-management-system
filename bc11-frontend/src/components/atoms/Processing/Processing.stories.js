import React from "react";
import Processing from "./Processing";

export default {
  title: "Atoms/Process",
};

export const Process = () => {
  return <Processing />;
};
