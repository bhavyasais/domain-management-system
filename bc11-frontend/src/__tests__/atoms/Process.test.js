jest.mock("react-lottie");
import React from "react";
import renderer from "react-test-renderer";
import Processing from "../../components/atoms/Process/Processing";

describe("<Processing/>", () => {
  it("Renders correctly", () => {
    let wrapper = renderer.create(<Processing />);
    expect(wrapper).toMatchSnapshot();
  });
});
