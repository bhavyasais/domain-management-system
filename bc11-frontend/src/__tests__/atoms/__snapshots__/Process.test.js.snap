// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Processing/> Renders correctly 1`] = `<div />`;
